src/_reset.scss

```diff
@@ -0,0 +1,135 @@
+/* http://meyerweb.com/eric/tools/css/reset/ 
```

Comment:
reset - is usually stored in the project as reset.css (not reset.scss) so that we can import it straightaway inside index.html.